### PR TITLE
Update readmes

### PR DESCRIPTION
**Pull request by ericliu121187**
# Short Deck Poker
*Also known as Six Plus Hold'em*

### What is it?

Phil Ivey

Most people who play poker know who Phil Ivey is. Most people who play poker also know what Texas Hold'em is.

Not everyone knows about Short Deck Poker -- the newest form of poker to hit the felt.

Don't worry, we're here to bring it to you. By we, we mean us and Phil Ivey.

What is Short Deck Poker? It's a variation of Texas Hold'em with all the 2s - 6s removed.

Short Deck Poker

What does this mean? It means that there's a lot of action. It creates big pots. There is a lot of skill involved and there's a lot of luck involved.

You know what? Let's let Phil tell you himself!

Sponsored by Phil Ivey*

### Is this for real money?

Absolutely, well, yes and no. Initially, we'll only be accepting a cryptocurrency called JokerPokerCoin (JPC).

### What is JokerPokerCoin (JPC)?

JPC is a cryptocurrency 3.0, meaning what? It's a multilayered coin designed to flicker between different chains, mostly utilizing the blockchain and brickchain. No more proof of stake or proof of work. JPC utilizes NO PROOF. JPC does not keep records or histories of transactions. Why not? Instead of wasting time, energy, and resources on this, all that really matters if you think about it is the amount of JPC you have **right now**. And that's what JPC does, everyone keeps track of the amount everyone else has **right now**.

Trying to make a quick transfer to a fellow SDP player? A transaction for this will be instantaneous on the blockchain.

Trying to deposit millions so you can play against Ivey himself? A transaction for this will require more confirmations on the brickchain and may take up to 20-25 seconds. Why the delay? There's more security involved with a certain amount of error prevention and rollback built in.

### Where can I get JokerPokerCoin (JPC)?

We're currently finalizing the proposal on how to release it to the public. When that's done, the players who signed up for the beta release of Short Deck Poker will be granted an amount of JPC (TBD).

### This sounds great. Let me play!

Right now, we're accepting a short list of beta testers who will also be granted a number of JPC. To get on this list, please email me at ericliu121187@gmail.com.

### How can I contribute to this project?

Currently, we're mainly looking for people with marketing and go-to-market product experience.

We're not really looking for software engineers as we've been getting a lot of traction in the tech scene and the applications having been flying in.

That being said, if you're a talented individual, feel free to shoot me a message at ericliu121187@gmail.com. Because we all know talent beats hard work.

## Roadmap
- Scale number of tables due to increased demand
- Launch and integrate JokerPokerCoin cryptocurrency

## Team
- [Eric Liu](https://github.com/ericliu121187)
- [Teddy Lo](https://github.com/teddylo57)
- [Tom Xu](https://github.com/txu89)

**Sponsorship with Phil Ivey is not finalized.*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,65 @@
 # Short Deck Poker
+*Also known as Six Plus Hold'em*
+
+## Overview
+
+### What is it?
+
+![Phil Ivey](assets/phil_ivey.png)
```
Comment (txu89) [inline README.md]: Just a link but no image.

File: README.md [hunk 2]
```diff
@@ -1 +1,65 @@
...
+
+![Short Deck Poker](assets/sdp.png)
+
+### How is Short Deck Poker (SDP) different from regular Texas Hold'em?
+
+It means that there's a lot of action. It creates big pots. There is a lot of skill involved and there's a lot of luck involved.
+
+You know what? Let's let Phil tell you himself!
+
+![Sponsored by Phil Ivey](assets/phil_ivey_sdp.png)*
```
Comment (txu89) [inline README.md]: This image also doesn't work.

File: README.md (hunk 1 above)
Comment (txu89) [reply README.md]: Trying changing URL to https://raw.githubusercontent.com/ericliu121187/short-deck-poker/1d918a0c96ee5b6ad0b8b4706f1f0ce56a81ddc2/assets/phil_ivey.png

**Comment by ericliu121187** [comment]
damn i followed all the sites and this site does it look! https://github.com/mark-anders/relative-image-url

can we try merging it and i'll fix right away if it doesn't work?

**Pull request merged**